scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java

```diff
@@ -97,14 +105,18 @@ public Scheduler login(Credentials cred) throws LoginException, AlreadyConnected
         String user = unPrincipal.getName();
 
         logger.info("user : " + user);
+
+        // get AWS S3 credentials
+        getAWSCredentials();
```

Comment:
authentication to the scheduler is not related to authentication with S3. Two different concepts